Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -60,7 +45,26 @@ public static int FindDigit(string equation)
             {
                 ans = divide1(expression1, expression3, expressions[1]);
             }
-            return ans;          
+            return ans;          
+        }
+        public static int[] checkExpression(int[] initialize,string[] expressions){
+            int k = 0;
+            for (int i = 0; i < expressions.Length; i++)
+            {
+                String expression = expressions[i];
+                int flag = 0;
+
+                if (expression.Contains('?'))
+                {                                                   //if the string doesn't contains '?' than we convert the string into integer
+                    flag = 1;                                       // and stores it in array
+                    initialize[k++] = -1;
+                }
+                if (flag == 0)
+                {
+                    initialize[k++] = Int32.Parse(expression);            
+                }                                                   
+            }
+            return initialize;
         }
         public static int multiply1(int expression1, int expression2, string original)

```

Comment:
Why is the method name multiply1, why not just multiply?
This method seems to be doing a lot more than just multiplying. Name it accordingly or break it into multiple smaller methods.